dipy/direction/peaks.py

```diff
@@ -679,11 +680,13 @@ def peaks_from_positions(
     position : array, (N, 3)
         Voxel coordinates of the N positions.
```

Comment:
This description seems to be wrong since we have the affine now. 

It should be `World coordinates`. Can you check and correct.

Reply:
Good catch. Fixed, thanks.